src/utils/privacy.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-len */
```

Comment:
Use the `ignoreStrings` option in the `max-len` config instead of having to disable `max-len` for long strings in the code itself.